src/hooks/useValidate/useValidate.ts

```diff
@@ -0,0 +1,28 @@
+import { usePhoneValidation } from 'react-international-phone'
+import { getDefaultCountries } from '../../atoms/phoneInputField'
+
+interface UseValidateHook {
+  isPossibleE164Number: (number: string) => boolean
+  isValidE164Number: (number: string) => boolean
+}
+
+export const useValidate = (): UseValidateHook => {
```

Comment:
Make sure to export this hook in `src/hooks/index.ts` and `src/hooks` as it might be useful for customers who would like to build a custom implementation.

Reply:
Done